src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,83 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_user_isOK() {
+        User expected = new User("Ivan2004", "1234567", 20);
+        User actual = registrationService.register(expected);
+        assertEquals(actual, expected);
+    }
+
+    @Test
+    void register_userPassword_isNotOK() {
```

Comment:
```suggestion
    void register_shortPassword_notOk() {
```
second part of name should be about given conditions, what is wrong with password? also, test length of 5, it's an edge case
third part is about overall result of the test